wire-schema/src/commonMain/kotlin/com/squareup/wire/schema/Field.kt

```diff
@@ -45,7 +45,7 @@ data class Field(
 
   val default: String?,
 
-  private val elementType: String,
+  private var elementType: String,
```

Comment:
Making this mutable feels bad. But not worse than the rest of linking, which is also very mutable.